client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Transports.ts

```diff
@@ -247,12 +247,12 @@ export class LongPollingTransport implements ITransport {
         }
 
         this.pollXhr = pollXhr;
-        this.pollXhr.open("GET", url, true);
+
+        this.pollXhr.open("GET", `${url}&_=${Date.now()}`, true);
         if (transferMode === TransferMode.Binary) {
             this.pollXhr.responseType = "arraybuffer";
         }
-        // IE caches xhr requests
-        this.pollXhr.setRequestHeader("Cache-Control", "no-cache");
```

Comment:
Keep this?

Reply:
why? it's not needed anymore and because the url is now different every time nothing on the way should cache anything either.

Reply:
OK